src/ui/page/home.html

```diff
@@ -1,56 +1,72 @@
 <Page title="overview">
-    <div class="grid-row">
-        <div class="column-full">
-            <h2 class="heading-medium basket__heading">Welcome back, {{signedInUser.givenName}}</h2>
+    <div class="container">
+        <div class="grid-row">
+            <div class="column-full">
 
-            <h3 class="heading-small">Required learning</h3>
+                <h2 class="heading-medium heading heading--medium heading--red-top">Required learning</h2>
 
-            {{#if requiredLearning.length}}
-            <ul>
-                {{#each requiredLearning as course}}
-                <li>
-                    <Learning bind:course />
-                </li>
-                {{/each}}
-            </ul>
-            {{else}}
-            <p>You have completed all your required learning.</p>
-            {{/if}}
-
-            <h2 class="heading-small">Learning plan</h2>
+                {{#if requiredLearning.length}}
+                <ul class="resource__list">
+                    {{#each requiredLearning as course}}
+                    <li class="resource">
+                        <Learning bind:course />
```

Comment:
I realise this was in a previous PR, but why are we doing `bind:course` here?

Reply:
I don't think I made this change, although I did notice when I was doing some of the Breadcrumb code, whenever I `<Breadcrumb :breadcrumbs/>` prettier would sometimes recompile to  `<Breadcrumb:breadcrumbs/>` and the server couldn't render the template. Maybe `bind:` bypasses it? unsure, doesn't make a difference though right? I can change it though

Reply:
Unless I've misunderstood `bind`, I think `course={{course}}` would be preferable?